tools/clang/test/CodeGenSPIRV_Lit/vk.location.double4.error.hlsl

```diff
@@ -0,0 +1,12 @@
+// RUN: not %dxc -T ps_6_0 -E main -fcgl  %s -spirv 2>&1 | FileCheck %s
```

Comment:
@python3kgae Added you here for review since this is the first lit test I've added and it's an expected error. I noticed that `not .... 2>&1 |` was how you handled it for some of the SPIR-V tests but it doesn't look like the DXIL Validation lit tests have these extra steps to redirect stderr. Is there a LIT configuration we can take advantage of to do the same thing for SPIR-V or is this the "right way"?

Reply:
Example: https://github.com/microsoft/DirectXShaderCompiler/blob/main/tools/clang/test/CodeGenHLSL/check_warning.hlsl#L1

Reply:
If the error can be generated in Sema rather than CodeGen (which is generally preferrable), you can write the test using the DXC `-verify` flag instead of FileCheck.

The clang verifier has the advantage of verifying the source locations of the error. Also performing validations in Sema allows them to be identified by the libClang-based intellisense plugin so that the errors can be surfaced in editor environments.